src/app/wallet/wallet/receive/receive.component.html

```diff
@@ -58,7 +58,7 @@
           <mat-form-field class="icon-input label-input full-width">
             <input matInput class="label" type="text" value="{{ selected?.label }}" [ngClass]="{'no-label': selected?.label === '(No label)'}" [disabled]="addressInput" [(ngModel)]="label" (keyup.enter)="updateLabel(getUnusedAddress().address)">
             <!-- TODO: default: -->
-            <mat-icon matTooltip="Edit label" *ngIf="addressInput" fontSet="partIcon" fontIcon="part-pen-1" (click)="addressInput = !addressInput"></mat-icon>
+            <mat-icon matTooltip="Edit label" *ngIf="addressInput" fontSet="partIcon" fontIcon="part-pen-1" (click)="emptyLabel();addressInput = !addressInput"></mat-icon>
```

Comment:
If we are creating a function for handling the click event, then the additional actions (the `addressInput = !addressInput` part in this case), should be moved inside of the function and the function name can then be changed to something more meaningful.